frontend/src/lib/lemon-ui/Popover/Popover.tsx

```diff
@@ -120,7 +123,7 @@ export const Popover = React.forwardRef<HTMLDivElement, PopoverProps>(function P
                 apply({ availableWidth, availableHeight, rects, elements: { floating } }) {
                     Object.assign(floating.style, {
                         maxHeight: `${availableHeight}px`,
-                        maxWidth: `${availableWidth}px`,
```

Comment:
So, the reason this is potentially dangerous is now we're not taking `availableWidth` into account – and we _always_ should be. But if we just `width` instead of `max-width` in `.new-insight-overlay`, this'd work well – that width would be used most of the time, but `max-width` would automatically take precedence when needed.

Reply:
🤔